docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1079,6 +1079,157 @@ These are not currently modeled.
 #. There is some evidence that young children are missed in the post enumeration survey and therefore are missed more than accounted for here [OHare_2019]_ 
 #. It is assumed that race and age/sex are independent, do not have interaction, and combine additively 
 
+Household Surveys
+^^^^^^^^^^^^^^^^^
+
+There are many different types of household surveys that we might want to include 
+in the model. Therefore, this documentation reflects a general framework for 
+household surveys. The research team would then be responsible for providing the 
+additional inputs for an individual survey. 
+
+**When to Sample** 
+
+There are two types of sampling plans: 
+
+1. A new random sample of the population at a defined time interval (e.g., sample 5% of the households each month for a year)
+
+- The sample will be taken at set time intervals (monthly, annually) that the research team will communicate in terms of time steps 
+- The total duration of the survey will be specified 
+- Each subsequent sample is assumed to be independent of the prior samples 
+
+2. A longitudinal sample will sample the SAME population on defined time steps (e.g., sample the same 5% of households each month for 6 months) 
+
+- The sample will be taken at set time intervals (monthly, annually) that the research team will communicate in terms of time steps 
+- The total duration of the survey will be specified 
+- Once there are stable addresses, the sample will be taken and kept the same at the address level. For now, when addresses are not stable, we can keep the same households instead. 
+
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - Home Address 
```

Comment:
Might just be zip code sometimes, can anticipate by including zip code as a separate field.